src/credentials-manager/la-policies.js

```diff
@@ -0,0 +1,8 @@
+export default {
+  deviceOwnerAuthenticationWithBiometrics: 1,
+  deviceOwnerAuthentication: 2,
+  deviceOwnerAuthenticationWithWatch: 3,
+  deviceOwnerAuthenticationWithBiometricsOrWatch: 4,
+  deviceOwnerAuthenticationWithWristDetection: 5,
```

Comment:
As mentioned above, these should not be supported because this SDK does not support macOS nor watchOS.